Library/Formula/multitail.rb

```diff
@@ -1,16 +1,26 @@
-require 'formula'
-
 class Multitail < Formula
-  homepage 'http://www.vanheusden.com/multitail/download.html'
-  url 'http://www.vanheusden.com/multitail/multitail-6.2.1.tgz'
-  sha1 '30bd77c4f542cd6ad6f1a0351f7d9e0a6cef6585'
+  homepage "http://vanheusden.com/multitail/"
+  url "http://www.vanheusden.com/multitail/multitail-6.4.1.tgz"
+  sha1 "9191b807f8d727810b2824a9bc8aafa17cb0d10e"
+
+  # Upstream pull request to fix compilation issues on OS X
+  # https://github.com/flok99/multitail/pull/13
```

Comment:
Nice work! Let's use the commit URL now it's merged: https://github.com/flok99/multitail/commit/65a329.diff